### PR TITLE
Adding more foods and processes Including Kebabs.

### PR DESCRIPTION
**Pull request by bioastroiner**
Yea, more Foods and more Chains to Come...
things i mainly ranted about over discord, so i keep it up to date here as well
image
image

### CODE REVIEW

**Comment by bioastroiner** [comment]
New Things so far just a sum up
Brand New Fat Process:
image
image
Brand New Kebabs [WIP]
image

image
yes and textures are more gregy from now on

**Comment by bioastroiner** [comment]
Here's what this PR has achieved so far
image
Meat Processing
image
Fat & Sludge Processing (this is old recipe it's fixed now)
image
image

- added Fat, can be processed into Soap,Food & Fuel, extracted from Animal Meat or Saturating Plant Oil with Hydrogen
- A Brand New Line of Foods: Kebabs! there was no way i gonna cover them all but did some basics
- Fat is now Valid for BioDiesel
- Sludge has a Treatment Propose now other than just making Chum! making Biomass and...

File: src/main/java/gregtechfoodoption/item/GTFOMetaItem.java [hunk 1]
```diff
@@ -434,6 +459,43 @@ public void registerSubItems() {
         HOT_RABBIT_STEW = addItem(133, "food.rabbit_stew.hot").addComponents(new GTFOFoodStats(9, 0.9f, false, false, new ItemStack(Items.BOWL))
                 .setEatingDuration(60));
 
+        SKEWER = addItem(135, "component.skewer");
+        KEBAB_KUBIDEH = addItem(136, "food.kebab.kubide").addComponents(new GTFOFoodStats(3, 0.2f, false, false, SKEWER.getStackForm(1))
```
Comment (bruberu) [inline src/main/java/gregtechfoodoption/item/GTFOMetaItem.java]: Consider adding negative effects to uncooked foods, or make them not consumable.

File: src/main/java/gregtechfoodoption/item/GTFOMetaItem.java [hunk 2]
```diff
@@ -434,6 +459,43 @@ public void registerSubItems() {
         HOT_RABBIT_STEW = addItem(133, "food.rabbit_stew.hot").addComponents(new GTFOFoodStats(9, 0.9f, false, false, new ItemStack(Items.BOWL))
                 .setEatingDuration(60));
 
+        SKEWER = addItem(135, "component.skewer");
```
Comment (bruberu) [inline src/main/java/gregtechfoodoption/item/GTFOMetaItem.java:462]: I usually keep my components above the foods to keep things organized.

File: src/main/java/gregtechfoodoption/recipe/chain/KebabChain.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                .inputs(KEBAB_KUBIDEH_COOKED.getStackForm(5))
+                .inputs(BURGER_CHUM.getStackForm(14))
+                .input(Items.BUCKET)
+                .fluidInputs(RabbitStew.getFluid(250))
+                .fluidInputs(HotFryingOil.getFluid(250))
+                .fluidInputs(PotatoJuice.getFluid(4000))
+                .outputs(KEBAB_CHUM_BUCKET.getStackForm(),SKEWER.getStackForm(7))
+                .buildAndRegister();
+
+        GTFOUtils.addBakingOvenRecipes(KEBAB_KUBIDEH.getStackForm(),KEBAB_KUBIDEH_COOKED.getStackForm(),1000,800,4);
```
Comment (bruberu) [inline src/main/java/gregtechfoodoption/recipe/chain/KebabChain.java]: Formatting can be improved by adding spaces after commas (you can use ctrl-shift-alt-L to do a whole file at once)

**Review by bruberu — COMMENTED** [review]
Looks good so far, although there are some things that should probably get improved.

**Review by bruberu — APPROVED** [review]
This looks pretty good now. If you like, you may keep adding more recipes and kebabs, or you can merge it now.

**Comment by bioastroiner** [comment]
yea this wraps for this PR, i keep later PRs shorter to more specefic groups of kebabs (or other foods), this one warpped Kubideh and Chum branch pretty much

**Pull request merged**